core/src/marker/markerManager.cpp

```diff
@@ -87,6 +87,9 @@ bool MarkerManager::setVisible(MarkerID markerID, bool visible) {
 }
 
 bool MarkerManager::setPoint(MarkerID markerID, LngLat lngLat) {
+
+    if (!m_mapProjection) { return false; }
```

Comment:
I'd prefer checking for `m_scene` in all these conditionals for consistency, though practically it shouldn't matter.